doctrine.rst

```diff
@@ -861,6 +861,83 @@ control behavior:
 
     The ``message`` option was introduced in Symfony 7.1.
 
+Mapped Route Parameters
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When many route parameters are used to find more than one entity, it is mandatory to use #[MapEntity] attributes and this can become cumbersome::
```

Comment:
```suggestion
When many route parameters are used to find more than one entity,
it is mandatory to use `#[MapEntity]` attributes and this can become cumbersome::
```